scarlet2/io.py

```diff
@@ -0,0 +1,83 @@
+import h5py
+import os
+import numpy as np
+import jax.numpy as jnp
+from scarlet2 import *
```

Comment:
Add `from scarlet2.io import model_to_h5, model_from_h5`,

Reply:
In the test_save_outputs.py?